.eslintignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
Will this ignore the subfolder node_modules as well?

Reply:
Yes. It will ignore all levels of nesting of mode_modules